sections/03_categorize.md

```diff
@@ -290,80 +290,95 @@ by allowing more efficient use of the costly practice of manual annotation.
 original training set for the machine learning model [@arxiv:1610.05820] - in
-this case, whether a person's record was present. There
-are solutions to this challenge. Training algorithms in a differentially private
-information from accurate answers in a machine learning model. There are
-solutions to this challenge. Training algorithms in a differentially private
-manner provides a limited guarantee that the algorithms output will be equally
-likely to occur regardless of the participation of any one individual. The limit
-is determined by a single parameter which provides a quantification of privacy.
-Simmons et al. [@doi:10.1016/j.cels.2016.04.013] present the ability to perform
-GWASs in a differentially private manner and Abadi et al. [@arxiv:1607.00133]
-show the ability to train deep learning classifiers under the differential
-privacy framework. Federated learning
-[@arxiv:1602.05629] and secure aggregations
+this case, whether a person's record was present. There are solutions to this
+challenge. Training algorithms in a differentially private information from
+accurate answers in a machine learning model. There are solutions to this
+challenge. Training algorithms in a differentially private manner provides a
+limited guarantee that the algorithms output will be equally likely to occur
+regardless of the participation of any one individual. The limit is determined
+by a single parameter which provides a quantification of privacy. Simmons et al.
+[@doi:10.1016/j.cels.2016.04.013] present the ability to perform GWASs in a
+differentially private manner and Abadi et al. [@arxiv:1607.00133] show the
+ability to train deep learning classifiers under the differential privacy
+framework. Federated learning [@arxiv:1602.05629] and secure aggregations
 [@url:http://proceedings.mlr.press/v54/mcmahan17a.html
 @url:https://eprint.iacr.org/2017/281.pdf] are complementary approaches that
 reinforce differential privacy. Both aim to maintain privacy by training deep
 learning models from decentralized data sources such as personal mobile devices
 without transferring actual training instances. This is becoming of increasing
-importance with the rapid growth of mobile health applications.
-However, training process in these approaches places constraints on the
-algorithms used and can make fitting a model substantially more challenging.
-
-Applying deep learning algorithms to this domain provides considerable
-opportunity as well as challenges - such as patient privacy - that cannot be
-ignored. Techniques that enable training on data without sharing the raw data
-may have a part to play. Those applying deep learning to the domain should also
-consider the potential of deep neural networks to inadvertently leak the
-training data of participants. Training within a differential privacy framework
-may often be warranted.
+importance with the rapid growth of mobile health applications. However,
```

Comment:
This concluding sentence needs some wordsmithing. It looks like you only changed whitespace, but since your changes hit these lines can you take a stab at it? Otherwise, please add a `TODO` so we will come back to it.